src/reducers/versions.tsx

```diff
@@ -425,21 +426,20 @@ export const getParentFolders = (path: string): string[] => {
   return parents;
 };
 
-export const createInternalVersionFile = ({
-  file,
-  path,
-}: {
-  file: ExternalVersionFileWithContent;
-  path: string;
-}): InternalVersionFile => {
+export const createInternalVersionFile = (
+  file: ExternalVersionFileWithContent,
+): InternalVersionFile => {
   return {
     content: file.content,
     created: file.created,
     downloadURL: file.download_url,
+    filename: file.filename,
     id: file.id,
     isMinified: file.uses_unknown_minified_code,
-    sha256: file.entries[path] ? file.entries[path].sha256 : null,
+    mimeType: file.mimetype,
+    sha256: file.sha256,
     size: file.size,
+    type: file.mime_category,
```

Comment:
mmm, is `type` the same thing as `mime_category`?  - at a glance it'd be confusing that they were the same thing (to me, anyway).  But I'm not familiar with the possible values.

Reply:
It's true that `type` isn't such a great name, but it's been there for quite some time. It's like the "type" of file, either "text", "binary" or "image". Maybe I should change it to `fileType`?